packages/common/src/server.ts

```diff
@@ -94,7 +94,7 @@ export type YogaServerOptions<
           ? [serverContext?: TServerContext | undefined]
           : [serverContext: TServerContext]
       ) => GraphiQLOptions | false)
```

Comment:
Hope you don't mind me commenting here directly! 

Should this be `GraphiQLOptions | boolean` for consistency?

Reply:
```suggestion
      ) => GraphiQLOptions | boolean)
```